### PR TITLE
Homepage styling: issue #72

### PR DESCRIPTION
**Pull request by kailasnadh790**
Fix https://github.com/hlxsites/famous-smoke-cigaradvisor/issues/72

Test URLs:
- Before: https://main--famous-smoke-cigaradvisor--hlxsites.hlx.page/cigaradvisor/drafts/Kailas/homepage
- After: https://feature-homepage--famous-smoke-cigaradvisor--hlxsites.hlx.page/cigaradvisor/drafts/Kailas/homepage

### CODE REVIEW

**Comment by kailasnadh790** [comment]
@bstopp This PR has global styling for buttons and Section headings as well.

**Comment by bstopp** [comment]
Was there not 2 issues being fixed with this one PR?

**Comment by kailasnadh790** [comment]
@bstopp I had captured them as tasks under one issue https://github.com/hlxsites/famous-smoke-cigaradvisor/issues/72 .
Let me know if I need to create separate issues and separate the PRs. Sorry for the confusion.

File: cigaradvisor/blocks/carousel/carousel.css [hunk 1]
```diff
@@ -70,24 +73,31 @@
...
 }
 
-.carousel.block .carousel-arrow .next-icon {
-  transform: rotate(180deg);
-  margin: 10%;
+.carousel.block .carousel-arrow .next-icon::before {
+  content: '\f054';
+  font-family: var(--ff-fontawesome);
+  padding-left: 50%;
+  font-size: x-large;
```
Comment (bstopp) [inline cigaradvisor/blocks/carousel/carousel.css:90]: What does `x-large` mean? Let's be specific - either a variable or a px size (variable preferred if we have one)

File: cigaradvisor/blocks/carousel/carousel.js [hunk 2]
```diff
@@ -86,15 +86,15 @@ export default async function decorate(block) {
       if (currentIndex < (items.length - itemsToShow)) {
         currentIndex += 1;
         slidesWrapper.style.transform = `translate3d(-${currentIndex * offset}%, 0, 0)`;
-        block.querySelector('.arrow-prev').style.display = 'inline-block';
+        block.querySelector('.arrow-prev').style.display = 'flex';
```
Comment (bstopp) [inline cigaradvisor/blocks/carousel/carousel.js:89]: we shouldn't set literal styles on the elements. Should add a class which will set `display: none` as an override to the `display: flex` on the element.

File: cigaradvisor/styles/styles.css [hunk 3]
```diff
@@ -434,6 +443,74 @@ color: #b19b5e;
...
+  font-weight: 900;
+  color: #8e7b5c;
+  padding: 38px 15px;
+  font-size: 45px;
+}
+
+main .section>.default-content-wrapper>h2 ::before{
+  content: 'decor';
+    color: transparent;
+    background: url('/cigaradvisor/images/homepage/sectiontitle/media_10c44610cd58523f02bbd64b114069404ca081e41.png') center no-repeat;
```
Comment (bstopp) [inline cigaradvisor/styles/styles.css:459]: if it's in the CSS - it should probably come from the codebase, not from the media-bus

File: cigaradvisor/styles/styles.css [hunk 4]
```diff
@@ -434,6 +443,74 @@ color: #b19b5e;
...
+  padding : 10px 10px 40px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+main .button-container .button{
+  font-style: normal;
+  font-family: var(--ff-montserrat);
+  font-weight: 700;
+  color: #fff;
```
Comment (bstopp) [inline cigaradvisor/styles/styles.css:484]: use variables

File: cigaradvisor/styles/styles.css [hunk 5]
```diff
@@ -142,6 +142,7 @@
   --light-grey: #d3d3d3;
   --dark-grey: #3c3a3b;
   --off-white: #f7f7f7;
+  --section-background: #eaeaea;
```
Comment (bstopp) [inline cigaradvisor/styles/styles.css:145]: What color is this? `section-background` isn't clear - in the context of the use (below) it's for a section with a grey background, let's make that the variable name.

**Review by bstopp — CHANGES_REQUESTED** [review]

**Review by bstopp — APPROVED** [review]

**Pull request merged**